acct-user/snapclient/snapclient-0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the Snapcast client component"
+ACCT_USER_ID=115
+ACCT_USER_HOME=/var/lib/snapclient
```

Comment:
Please read 
https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html#choosing-a-home-directory
and decide whether you really need this to be set.

Reply:
snapclient doesn't appear to use the home dir, so it's probably safe to remove. However, the upstream Debian package does create it.